rpm/gratia-probe.spec

```diff
@@ -330,10 +336,10 @@ install -d $RPM_BUILD_ROOT/%{_sysconfdir}/gratia
 
 
 # Burn in the RPM version into the python files.
-grep -rIle '%%%%%%RPMVERSION%%%%%%' $RPM_BUILD_ROOT%{_datadir}/gratia $RPM_BUILD_ROOT%{python_sitelib} | while read file; do \
-  perl -wpi.orig -e 's&%%%%%%RPMVERSION%%%%%%&%{version}-%{release}&g' "$file" && \
-    rm -fv "$file.orig"
-done
+rpmver=%{version}-%{release}.${git_commit_id:0:7}
+find $RPM_BUILD_ROOT%{_datadir}/gratia $RPM_BUILD_ROOT%{python_sitelib} \
+  -type f -exec fgrep -ZIle '%%%%%%RPMVERSION%%%%%%' {} + | xargs -0 \
```

Comment:
I'm afraid to ask why you need 6 `%` on each side here but 3 `%` on each side in Gratia.py

Reply:
Oh right, specfile macros. never mind.